conbench/app/_plots.py

```diff
@@ -27,11 +27,11 @@ class BokehPlotJSONOrError:
 
 
 class TimeSeriesPlotMixin:
-    def get_outliers(self, benchmarks):
+    def get_biggest_changes(self, benchmarks):
```

Comment:
Renamed to avoid confusion from different things with the same name. Happy to rename to something else if y'all prefer

Reply:
cool, let's iterate on names over time.